Dz2.py

```diff
@@ -0,0 +1,10 @@
+i = (input("Введите текст: "))
+if (i.isdigit()) == True:
+    I = int(i)
+    if I % 2 == 0:
+        print("Вы ввели парное число")
+    else:
+        print("Вы ввели не парное число")
+else:
+    x = (len(i))

```

Comment:
There is no need to use parentheses around `len(i)`
And you can avoid the creation of `x` by putting `len(i)` directly into string instead of x